.circleci/config.yml

```diff
@@ -179,19 +181,27 @@ commands:
         type: string
     steps:
       - attach_workspace:
-          at: ~/mattermost-mobile
-      - deploy:
+          at: .
+      - run:
           name: "Deploy ipa to TestFlight"
           working_directory: fastlane
-          command: bundle exec fastlane ios deploy ipa:./<<parameters.ipa_path>>
+          command: bundle exec fastlane ios deploy ipa:$HOME/mattermost-mobile/<<parameters.ipa_path>>
```

Comment:
deploy-ios and deploy-android are almost the same, wouldn't you use a single deploy command with a target parameter where you especify `ipa:$HOME/mattermost-mobile/<<parameters.ipa_path>>` or `apk:$HOME/mattermost-mobile`

also if you do this you'd have to remove the attach workspace from the command, which might be a good idea anyways, as I'd prefer to separate what needs to generate the build environment from the steps of building